reference/docs-conceptual/dev-cross-plat/create-cmdline-predictor.md

```diff
@@ -0,0 +1,234 @@
+---
+description: This article describes how to create a command-line predictor to help with command completion in PowerShell.
+ms.date: 03/28/2022
+title: How to create a command-line predictor
+---
+# How to create a command-line predictor
+
+PSReadLine 2.1.0 introduced the concept of a smart command-line predictor by implementing the
+Predictive IntelliSense feature. PSReadLine 2.2.2 expanded on that feature by adding a plugin model
+that allows you create your own command-line predictors.
+
+Predictive IntelliSense enhances tab completion by providing suggestions, on the command line, as
+you type. The prediction suggestion appears as colored text following your cursor. This enables you
+to discover, edit, and execute full commands based on matching predictions from the user's history
+or additional domain-specific plugins.
```

Comment:
In this context, `you` is referring to the person using the Predictive IntelliSense, not (necessarily) the person _developing a predictor_, right?
```suggestion
to discover, edit, and execute full commands based on matching predictions from your command
history or additional domain-specific plugins.
```